pkg/cassandra/config/config.go

```diff
@@ -163,6 +163,10 @@ func (c *Configuration) NewCluster(logger *zap.Logger) (*gocql.ClusterConfig, er
 	return cluster, nil
 }
 
+func (c *Configuration) Close() error {
```

Comment:
Since this function did not exist before, nothing is calling it, so just adding it here is not going to address the issue in the ticket.

Reply:
So I looked at the code base, the leaky go routine only happens when new Cassandra Cluster is created directly with no option to close. So, the leaky-go routine is happening on our side, not on your end. I am thinking about changing a ticket from bug to feature request so that users who created the new Cassandra Cluster will have the option to close the fswatcher leaky go routine.